### ISSUE TITLE
Not poketed Faucet reward

### ISSUE THREAD

**Issue by ivijulsc** [clemahieu/raiblocks#134]
Hello, i was searching for this problem but i couldn't find information anywhere.
It's the first time i get to the Top 60 buy i don't see the transaction on my wallet balance.
I got this [http://i.imgur.com/zNICw5i.png](url)
I'm prertty sure that is a problem with blocks or something but i can't find the solution.
I allready read this Wiki [https://github.com/clemahieu/raiblocks/wiki/Troubleshooting-and-Missing-transactions](url) and still couldn't find a solution.
Do i need to wait? Do i need to configure something in my wallet?
I'm pretty sure that this is not the section to ask this but i couldn't find any place to ask.
Thanks.

**Comment by ivijulsc** [comment]
@clemahieu Can you tell me if it's normal that it take this long to syncronize?
It's been 24 hours with the rai wallet running and still it's not sync. I allready updated to 7.8.5
This is the status of my wallet 24 hours opened http://i.imgur.com/oE9bkbz.png
The number inside () goes up and down.

**Comment by clemahieu** [comment]
That's not normal.  It could be something firewall or antivirus related, do you have either of those?

Can you send you logs https://github.com/clemahieu/raiblocks/wiki/Log-files to https://www.dropbox.com/request/Jd81tYniIGRr2tTY6fdR

**Comment by ivijulsc** [comment]
@clemahieu I have my antivirus (avast) inactive and firewall deactivated. Just sended all my log files to you.
Thanks for answering.

**Comment by clemahieu** [comment]
The log is filled with 
[2017-05-27 19:41:39.852001]: UDP Receive error: Se ha forzado la interrupciÛn de una conexiÛn existente por el host remoto

I've seen this UDP connection error happen on Windows informing the application that a firewall dropped the packet.  Is the wallet permitted by the windows firewall?

You can also chat on slack.raiblockscommunity.net in #support

**Comment by ivijulsc** [comment]
I have firewall deactivated http://i.imgur.com/tgpl0sG.png
Im going to uni now, will let the wallet open and see if something happens 2 hours from now.
How long do you think it shuld take to fully sync? Few hours?
Thanks agian.

**Comment by ivijulsc** [comment]
2 Hours been passed and still getting the errors:
[2017-05-30 20:07:25.998721]: UDP Receive error: Se ha forzado la interrupción de una conexión existente por el host remoto
[2017-05-30 20:08:37.696735]: UDP Receive error: Se ha forzado la interrupción de una conexión existente por el host remoto
[2017-05-30 20:11:08.914384]: UDP Receive error: Se ha forzado la interrupción de una conexión existente por el host remoto

I really don't know what can be causing this. Firewall deactivated and Antivirus deactivated too.
Any other idea in what could be causing this?
Thanks

**Comment by newandynew** [comment]
@clemahieu I have the same problem with 7.8.4 wallet synchronization as ivijulsc. After 2 days online it still show "Block: XX (XXXXXX)". Number of blocks slowly encrease, values in () increase or decrease. My antivirus is disabled, and my firewall have permission to work with the wallet. However, I do not see in the log the same error messages as in the ivijulsc's log. I can send you my log for check. Please send me a link for upload.

**Comment by clemahieu** [comment]
Synchronizing was one of the improvements with 7.8.5. Try that version first.l, especially if you don't have this error in your log.

**Comment by newandynew** [comment]
Thanks fo advise, I will try it. Please advise if I should uninstall the current wallet before installation of another version.

**Comment by newandynew** [comment]
@clemahieu I had unistalled old wallet and installed 7.8.5 wallet. Nothing has changed. No problems with antivirus and firewall, but still no synchronization. I see again "Block XX (XXXXX)", with values in () slowly increasing or decreasing. HDD is working like crazy non-stop. I closed the wallet, no sense to continue. The log is attached.
[log_2017-06-01_04-47-00.2.zip](https://github.com/clemahieu/raiblocks/files/1043446/log_2017-06-01_04-47-00.2.zip)
I'll appreciate any suggestions on what can be done.

**Comment by icarusglider** [comment]
Running Windows 10, same trouble. Checked in resource monitor, all the outgoing connections are to IPv6 addresses, not a single IPv4 except to localhost. I am not on a IPv6 network, and get repeated UDP errors and EXTREMELY slow sync (8 blocks in 2 hours)

**Comment by rhaesa** [comment]
Does any of you have this issue resolved? I had the same problem and I just basically gave up. Is it possible to just send the xrb to another wallet?

**Comment by icarusglider** [comment]
@rhaesa please try using one of the bootstrap files from the Slack channel or try following the ramdisk workaround guide https://steemit.com/raiblocks/@icarusglider/raiblocks-wallet-sync-workaround

_Sent from my Lge LGLS991 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by rhaesa** [comment]
Hi, i followed what was on the link that you sent me but when I opened
raiblocks, iI was fiven a different xrb account. I dont know what to do
now. Is it possible to just cancel and send it to my telegram xrb account?

**Comment by icarusglider** [comment]
A bootstrap file contains the seed to a different, unused account. You must export your seed first from your own wallet, then move the file into your folder. Once open, you may import your seed to restore your accounts.

**Comment by rhaesa** [comment]
Hi, my wallet is finally syncing but it is taking very slow. Will it still
continue syncing even if raiblocks will be on maintenance?

**Comment by hearthless** [comment]
i have multiple accounts inside my wallet...ive noticed that all the said accounts has desame seed..that is normal?..if ever i import my wallet seed it is possible that all the accounts are still there?

**Comment by MinoMino** [comment]
@rhaesa Yes. The network is unaffected by the faucet being in maintenance.

@hearthless Yes, it's normal. The seed deterministically creates the accounts, so if you import the seed into a fresh wallet, all you have to do is click the `Create account` button again and the accounts, with all the funds, should appear again in the same order as when you first created them.

**Comment by hearthless** [comment]
thank you so much...all is well now

**Comment by octopyth** [comment]
I have version 9.0 on Mac and I completed a transaction:
https://raiblocks.net/block/index.php?h=62E49B1FFDCE604A4BB0E03AA7C5BC48692BB33C3E1C4ED6810379A6CD5DC667
It takes forever to show up in my wallet:
![image](https://user-images.githubusercontent.com/640759/34587005-1d89d7ba-f1af-11e7-9117-42cdbf4688d5.png)
When I try to search for this block in the advanced/block_viewer, I don't find the transaction.
What do I need to do to complete the transaction?
